CHANGELOG.txt

```diff
@@ -165,6 +165,7 @@
         # In JavaScript :
             - A class 'Assert()', which factorizes some validation code, has been added.
             - A builtin method 'instance' has been added in 'creme.utils.newJQueryPlugin'.
+            - title property of creme.dialog.Dialog is now unescaped (see bug #64)
```

Comment:
As the bugfix is backported to 2.1, this fix should not appear here (but in the 2.1.6's one)